pkg/scheduler/scheduler.go

```diff
@@ -315,6 +315,24 @@ func (s *Scheduler) getPlacement(resourceBinding *workv1alpha1.ResourceBinding)
 	return placement, string(placementBytes), nil
 }
 
+func (s *Scheduler) getClusterPlacement(crb *workv1alpha1.ClusterResourceBinding) (policyv1alpha1.Placement, string, error) {
```

Comment:
Nice job for adding a func `getClusterPlacement`.

Reply:
I'm glad to provide some help if needed.🧐 just cc me.

Reply:
This function only servers for `ClusterResourceBinding`, what about the `ResourceBinding`?